src/Tuttifrutti/Servant.hs

```diff
@@ -0,0 +1,53 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+module Tuttifrutti.Servant where
+
+import           Tuttifrutti.Prelude
+
+import qualified Network.Wai         as Wai
+
+import           Servant             (Handler (..), HasServer, ServerT, hoistServer, runHandler,
+                                      serve)
+
+import           Tuttifrutti.Http    (MonadHttp)
+import           Tuttifrutti.Log     (MonadLog)
+import           Tuttifrutti.Wai     (requestLogger, withXRequestId)
+
+-- | Serve an API with all possible fruits enabled out-of-box.
```

Comment:
Let's add info/links here about the fact that we have to specify the concrete types

Reply:
Fixed

Reply:
Lovely 👏